entity-view/impl/src/main/java/com/blazebit/persistence/view/impl/metamodel/ViewMappingImpl.java

```diff
@@ -94,6 +95,8 @@
     private ManagedViewTypeImplementor<?> viewType;
     private Map<EmbeddableOwner, ManagedViewTypeImplementor<?>> embeddableViewTypeMap;
 
+    private List<With> withAnnotations = new ArrayList<>();
```

Comment:
Null by default to avoid unnecessary allocations. Also, rename to `Set<CteProvider> cteProviders`